tests/acceptance/features/webUIPreview/imageMediaViewer.feature

```diff
@@ -1,8 +1,17 @@
 Feature: display image in media viewer on the webUI
 
-  Scenario: preview of image with media viewer is possible
+  Background:
     Given user "user1" has been created with default attributes
-    And user "user1" has uploaded file "testavatar.jpg" to "testavatar.jpg"
+
+  Scenario: preview of image with media viewer is possible
+    Given user "user1" has uploaded file "testavatar.jpg" to "testavatar.jpg"
     And user "user1" has logged in using the webUI
     When the user views the file "testavatar.jpg" in the mediaviewer using the webUI
     Then the file "testavatar.jpg" should be displayed in the mediaViewer
+
+  Scenario: preview of video with media viewer is possible
+    Given user "user1" has logged in using the webUI
+    And a file with the size of "30000000" bytes and the name "big-video.mp4" has been created locally
```

Comment:
I think its better to have a real video here, just commit a real video to filesForUpload and then you can use it the same as in the pictures tests
e.g. use the big buck bunny video https://peach.blender.org/download/